test/e2e/components/vl-titles.js

```diff
@@ -1,9 +1,21 @@
 const { VlElement } = require('vl-ui-core').Test;
 
-class VlTitles extends VlElement {  
-    constructor(driver, selector) {
-        super(driver, selector);
+class VlTitle extends VlElement {
+    async isSansFont() {
+        return this.hasAttribute('sans');
+    }
+
+    async hasBorder() {
+        return this.hasAttribute('has-border');
+    }
+
+    async isAlt() {
+        return this.hasAttribute('alt');
+    }
+
+    async isH(headerNumber) {
+        return await this.getTagName() === `h${headerNumber}`;
     }
```

Comment:
Ik zou de functies voorzien voor de h1 tot de h6.

Reply:
Gaan we nu niet doen